samples/sampler/stories/default/PageViews.js

```diff
@@ -15,54 +15,155 @@ export default {
 	component: 'PageViews'
 };
 
-export const _PageViews = (args) => (
-	<PageViews arranger={BasicArranger} pageIndicatorType={args['pageIndicatorType']}>
-		<PageViews.Page aria-label="This is a description for page 1">
```

Comment:
Using a switch statement doesn't seem to be the appropriate way. I thought we would use the children(PageViews.Page) array.

Reply:
Could not find a way to use children array
because there is not any prop to control the number of page in PageViews component. 
Instead, I changed the sample code more readable.

Reply:
Could you please revert this change in the sample? It seems that controlling PageViews number does not work properly.

Reply:
OK. I reverted changes in the sample.